resources/views/layouts/_fathom.blade.php

```diff
@@ -1,5 +1,5 @@
-@production
+@env(['production', 'testing'])
     <!-- Fathom - beautiful, simple website analytics -->
-    <script src="https://boom.laravel.io/script.js" site="UXCUXOED" defer></script>
+    <script src="https://boom.laravel.io/script.js" site="UXCUXOED" defer @if(isset($canonical) && !Str::startsWith($canonical, config('app.url')))data-canonical="false"@endif></script>
```

Comment:
What's the downside of always applying `data-canonical="false"`?

Reply:
It falls back to `window.location` afaik which is not so reliable.

For instance, we give `https://laravel.io/forum` and `https://laravel.io/forum?page=1` the same canonical URL so that search indexes don't index them both an penalise us for duplicate content. If we don't conditionally add it, Fathom will treat both as separate pages.